.nginx.conf

```
@@ -1,6 +1,29 @@
+## Example configuration:
+# upstream php {
+#    # use tcp connection
+#    # server  127.0.0.1:9090;
+#    # or socket
+#    server   unix:/var/run/php/php7.2-fpm.sock;
+# }
+# server {
+#    listen 80;
+#    server_name forum.domain.tld;
+#    root /var/www/forum.domain.tld/public;
+#    include /var/www/forum.domain.tld/.nginx.conf;
+# }
+#
+
 # Pass requests that don't refer directly to files in the filesystem to index.php
 location / {
-  try_files $uri $uri/ /index.php?$query_string;
+  try_files $uri $uri/ /index.php?$args;
```

Is it not better to use `$is_args` instead of forcing the `?` to be a part of the forwarded URL?

```suggestion
  try_files $uri $uri/ /index.php$is_args$args;
```

---

You are totally right, I opened the PR before updating my own nginx configuration for WordPress 5.x, but it's definitively better to use `$is_args$args` insteand of  `?$args`